src/generators/crud/default/views/view.php

```diff
@@ -10,16 +10,17 @@
 
 /** @var \yii\db\ActiveRecord $model */
 /** @var $generator \schmunk42\giiant\generators\crud\Generator */
-
 ## TODO: move to generator (?); cleanup
-$model = new $generator->modelClass();
+$model          = new $generator->modelClass();
```

Comment:
Could you reformat the indents for variables assignment, will also point to some other...